exercises/practice/crypto-square/.docs/instructions.md

```diff
@@ -5,9 +5,9 @@ Implement the classic method for composing secret messages called a square code.
 Given an English text, output the encoded version of that text.
 
 First, the input is normalized: the spaces and punctuation are removed
-from the English text and the message is downcased.
+from the English text and the message is down-cased.
 
-Then, the normalized characters are broken into rows.  These rows can be
+Then, the normalized characters are broken into rows. These rows can be
```

Comment:
Single sentence per line, but approving.

Reply:
Good catch. I will make a note that this needs to be fixed in the problem-specifications repository.